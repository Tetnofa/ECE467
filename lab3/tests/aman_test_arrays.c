// Test array declarations and operations
int[5] arr;
int[3] nums;

void initArrays() {
    int i;
    i = 0;
    while (i < 5) {
        arr[i] = i * 2;
        i = i + 1;
    }
}

int sumArray(int[5] input) {
    int sum;
    int i;
    sum = 0;
    i = 0;
    while (i < 5) {
        sum = sum + input[i];
        i = i + 1;
    }
    return sum;
} 