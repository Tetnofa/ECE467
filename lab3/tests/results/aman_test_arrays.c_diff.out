3a4
> cannot print AST with parse errors
